uint8 valid
float32 indicated_airspeed
float32 temperature
